### PR TITLE
formatting, refactored main, made some changes to prefix system and ready

### PR DESCRIPTION
**Pull request by orvitpng**

### CODE REVIEW

File: src/commands/SyncCommand.ts [hunk 1]
```diff
@@ -67,7 +67,7 @@ export default class SyncCommand extends Command {
 
   private getSyncCompleteEmbed(
     author: User,
-    start: number,
+    _start: number,
```
Comment (CoreyShupe) [inline src/commands/SyncCommand.ts:70]: just remove this field if it's unused

File: src/main.ts [hunk 2]
```diff
@@ -3,14 +3,10 @@ import { config } from "./types/Config.ts";
 import SyncCommand from "./commands/SyncCommand.ts";
 import SynthesisClient from "./client.ts";
 
-const client = new SynthesisClient();
+const client = new SynthesisClient("!");
 
 client.commands.add(SyncCommand);
```
Comment (CoreyShupe) [inline src/main.ts:8]: should we do this in the synthesis client if we're refactoring main under a function like `registerSynthesisCommands`

File: src/main.ts [hunk 3]
```diff
@@ -3,14 +3,10 @@ import { config } from "./types/Config.ts";
...
 
-client.on("ready", () => {
-  console.log(`Ready! User: ${client.user?.tag}`);
-});
-
 client.connect(config.discord.secret, [
```
Comment (CoreyShupe) [inline src/main.ts:10]: maybe make a function in synthesis client like `start()` or just `connect()` inside of the SynthesisClient class, that way we can really clean up main

**Review by CoreyShupe — CHANGES_REQUESTED** [review]

File: src/commands/SyncCommand.ts (hunk 1 above)
Comment (DebitCardz) [reply src/commands/SyncCommand.ts:70]: Ah yeah I removed the process time from the embed but forgot to remove the parameter.

File: src/main.ts (hunk 2 above)
Comment (orvitpng) [reply src/main.ts:8]: Huh?

File: src/main.ts (hunk 3 above)
Comment (orvitpng) [reply src/main.ts:10]: ill probably do it like my bot's code, gimme a bit

File: src/main.ts (hunk 2 above)
Comment (CoreyShupe) [reply src/main.ts:8]: remove the raw client calls from main and put them inside of functions in the client class

Comment (orvitpng) [reply src/main.ts]: look at my new commit

**Comment by orvitpng** [comment]
@CoreyShupe here ya go

File: src/client.ts [hunk 4]
```diff
@@ -1,17 +1,22 @@
...
-    super({
-      prefix: "!",
-    });
+  constructor(options: CommandClientOptions) {
+    super(options);
+    this.commands.add(SyncCommand);
 
     this.config = JSON.parse(
       Deno.readTextFileSync("../secrets/config.json"),
-    ) as Config;
```
Comment (CoreyShupe) [inline src/client.ts:15]: does this not give a Type Error by removing this?

Comment (orvitpng) [reply src/client.ts:15]: you don't have to be verbose about it

**Pull request merged**